packages/cherry-pick-bot/test/cherry-pick.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import nock from 'nock';
+import {describe, it} from 'mocha';
+import * as assert from 'assert';
+import {parseCherryPickComment} from '../src/cherry-pick';
+
+nock.disableNetConnect();
+
+describe('parseCherryPickComment', () => {
+  it('parses a command comment', () => {
+    const branch = parseCherryPickComment('/cherry-pick feature-branch');
+    assert.strictEqual(branch, 'feature-branch');
+  });
```

Comment:
If you break the logic out for finding the `headSha` into a helper, perhaps you could test this in isolation with some fake commits, to improve coverage and to ensure we actually pick the appropriate head.